test/AWS.Logger.AspNetCore.Tests/TestScope.cs

```diff
@@ -24,7 +24,7 @@ public void MakeSureCanCreateScope()
             }
 
             Assert.Single(coreLogger.ReceivedMessages);
-            Assert.True(coreLogger.ReceivedMessages.Contains("[Information] MakeSureCanCreateScope: log  \r\n"), "Messages don't contain actual log message.");
+            Assert.True(coreLogger.ReceivedMessages.Contains("[Information] MakeSureCanCreateScope: log\r\n"), "Messages don't contain actual log message.");
```

Comment:
These tests will fail if run on a platform where `Environment.NewLine` is not `\r\n`

Reply:
Updated tests to use `Environment.NewLine` constant.